bridge-fungible-token/tests/utils/ext_sdk_provider.rs

```diff
@@ -22,10 +22,11 @@ pub async fn get_contract_message_script() -> (Vec<u8>, Bytes32) {
 }
 
```

Comment:
This whole file was renamed in the predicates repo to just be `builder.rs`. See [here](https://github.com/FuelLabs/bridge-message-predicates/blob/master/contract-message-test/tests/utils/builder.rs)